spec/middleware/rack/attack_spec.rb

```diff
@@ -94,6 +94,34 @@ def app
     end
   end
 
+  describe 'facility_locator/ip' do
+    subject { last_response.status }
+
+    let(:endpoint) { '/facilities_api/v1/ccp/provider' }
+    let(:limit) { 3 }
```

Comment:
Since 3 is the rate limit, should we set the test to something greater than 3, which would then throw a 429 (I think)?

Reply:
Ah nvm, I think line 109 is the fourth call to the endpoint. Scratch that